code-challenges.test.js

```diff
@@ -26,16 +26,53 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
-
+// const object1 = { number: 15 }
+// // Expected output: "15 is divisible by three"
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
+// // Expected output: "-7 is not divisible by three"
+
+describe("divisableByThree", () => {
+    it("takes a object decides if the number inside it is evenly divisible by three or not", () => {
+        expect(divisableByThree(object1.number)).toEqual("15 is divisible by three")
+        expect(divisableByThree(object2.number)).toEqual("0 is divisible by three")
+        expect(divisableByThree(object3.number)).toEqual("-7 is not divisible by three")
+    })
+})
```

Comment:
The prompt states the function should take an object as an argument. This test is passing a number as an argument.